ztp/du-profile-kube-compare/required/sriov-operator/SriovNetwork.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: {{ .metadata.name }}
+  namespace: openshift-sriov-network-operator
+spec:
```

Comment:
It would be OK here to simply take everything in ".spec" as allowable.

Reply:
Updated